archlinuxcn/sendanywhere/lilac.yaml

```
@@ -0,0 +1,13 @@
+maintainers:
+  - github: Xeonacid
+    email: h.dwwwwww@gmail.com
+
+build_prefix: extra-x86_64
+
+pre_build: aur_pre_build
+
+post_build: aur_post_build
+
+update_on:
+  - source: aur
+    aur: sendanywhere
```

```suggestion
  - source: manual
    manual: 1
```
@DuckSoft How about this?

---

uh, If i were correct, `manual` means lilac completely don't manage this package, so that the built package should be signed by a trusted maintainer and placed manually into repo.archlinuxcn.org...

---

No, `manual` simply means the version is manually filled. It's useful to lock to some version, or as a manual rebuild indicator.

---

There doesn't seem to be a version number somewhere outside the deb package....Weird upstream. There is no way for lilac to keep up automatically.

---

Oh, maybe use `httpheader` to check `last-modified` or `etag`?

```
>>> curl -I https://update.send-anywhere.com/linux_downloads/sendanywhere_latest_amd64.deb
HTTP/2 200 
content-length: 69265914
content-type: application/octet-stream
last-modified: Fri, 13 Nov 2020 03:25:17 GMT
accept-ranges: bytes
etag: "5fadfc9d-420e9fa"
x-cache: TCP_MISS
server: nginx/1.14.0 (Ubuntu)
...
```

---

@lilydjwg that was... weird... but interesting! Will you put this into nvchecker, i guess?

---

It's already there: https://nvchecker.readthedocs.io/en/latest/usage.html#search-in-an-http-header

---

uh, so how can nvchecker get the `Fri, 13 Nov 2020 03:25:17 GMT` version and compare with existing ones...

---

```toml
source = "httpheader"
url = "https://update.send-anywhere.com/linux_downloads/sendanywhere_latest_amd64.deb"
header = "last-modified"
regex = ".*"
```